src/views/my-account/my-account.view.jsx

```diff
@@ -23,6 +23,9 @@ import Button from '../shared/button/button.view'
 import { copyToClipboard } from '../../utils/browser'
 import { ReactComponent as QRCodeIcon } from '../../images/icons/qr-code.svg'
 
+const packagejson = require('../../../package.json')
```

Comment:
I don't like this `require` here, it would work because `create-react-app` adds polyfills for this but it would be better to change it to `import`